modules/debts_api/lib/debts_api/v0/fsr_form_transform/utils.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+# rubocop:disable Metrics/ModuleLength
```

Comment:
This is a great refactor. I have one suggestion that I think will work but you would want to test to confirm. 

Instead of `re_camel`, `re_camel_array`, and `re_camel_hash` you can combine those into one method and then remove the `rubocop:disable`

```ruby
def re_camel(x)
  case x
  when Array
    x.map { |el| re_camel(el) }
  when Hash
    x.transform_keys { |key| key.camelcase(:lower) }.transform_values { |val| re_camel(val) }
  else
    x
  end
end
```

Reply:
This is a great idea! I'll try it out.